static/manifests/calico/DaemonSet/calico-node.yaml

```diff
@@ -252,12 +252,12 @@ spec:
             preStop:
               exec:
                 command:
-                - /bin/calico-node
+                - /usr/bin/calico-node
```

Comment:
Upstream uses UBI based images where `/bin` is linked to `/usr/bin`. Now with 3.28.1 update, we build the image on alpine where this linking is not valid. Hence the patch changes.